src/app/components/named-entity-ref/named-entity-ref.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, Input } from '@angular/core';
+import { of } from 'rxjs';
+import { map, share } from 'rxjs/operators';
+import { NamedEntityRefData } from '../../models/evt-models';
+import { register } from '../../services/component-register.service';
+import { NamedEntitiesParserService } from '../../services/xml-parsers/named-entities-parser.service';
+
+@Component({
+  selector: 'evt-named-entity-ref',
+  templateUrl: './named-entity-ref.component.html',
+  styleUrls: ['./named-entity-ref.component.scss'],
+})
+@register
+export class NamedEntityRefComponent {
+  @Input() data: NamedEntityRefData;
+
+  entity$ = this.neParserService.namedEntities$.pipe(
+    share(),
```

Comment:
??

Reply:
I really don't know why I used it. It's probably a copy-paste from somewhere else. I'll remove it!